users/views.py

```diff
@@ -22,3 +23,10 @@ def register(self, request):
         serializer.save()
         return Response(serializer.data)
 
+    @list_route(methods=['post'], permission_classes=[AllowAny])
```

Comment:
Please create a new view. You don't need the decorator as we are Not using viewsets anymore.